xml2kinto/tests/test_main.py

```diff
@@ -1,8 +1,20 @@
+import codecs
+import json
 import mock
 import unittest
 
 from xml2kinto import __main__ as main
 
+SCHEMAS = {
+    'certificates': None,
+    'gfx': None,
+    'add-ons': None,
+    'plugins': None,
+}
+
+with codecs.open(main.SCHEMA_FILE, encoding='utf-8') as f:
+    SCHEMAS = json.load(f)
```

Comment:
I don't see a test needing a real schema, so I would drop those two lines and use the fake/empty schemas from above.